src/nl/rubensten/texifyidea/inspections/latex/LatexEnDashInspection.kt

```diff
@@ -24,12 +24,15 @@ open class LatexEnDashInspection : TexifyRegexInspection(
         groupFetcher = { listOf(it.group(2), it.group(3)) }
 ) {
 
-    override fun applyFix(project: Project, descriptor: ProblemDescriptor, replacementRange: IntRange, replacement: String, groups: List<String>) {
+    override fun applyFix(project: Project, descriptor: ProblemDescriptor, replacementRange: IntRange, replacement: String, groups: List<String>): Int {
         val file = descriptor.psiElement as PsiFile
-        val document = file.document() ?: return
+        val document = file.document() ?: return 0
         val start = groups[0]
         val end = groups[1]
 
-        document.replaceString(replacementRange.start, replacementRange.endInclusive, "$start--$end")
+        val dashReplacement = "$start--$end"
+        document.replaceString(replacementRange.start, replacementRange.endInclusive, dashReplacement)
+
+        return dashReplacement.length - (replacementRange.endInclusive - replacementRange.start)
```

Comment:
I see the structure `IntRange.endInclusive - IntRange.start` a lot.

I'd suggest a util in `General.kt` like `val IntRange.length: Int`

I actually use one a lot myself in my own projects. Apparently I haven't added it yet to TeXiFy...